### PR TITLE
Document MDC trace IDs

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

Updated quickstart app https://github.com/quarkusio/quarkus-quickstarts/pull/485

### CODE REVIEW

**Review by cescoffier — APPROVED** [review]

File: docs/src/main/asciidoc/opentracing.adoc [hunk 1]
```diff
@@ -65,13 +65,18 @@ import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```
Comment (gsmet) [inline docs/src/main/asciidoc/opentracing.adoc]: This should be changed to use JBoss Logging.

File: docs/src/main/asciidoc/opentracing.adoc [hunk 2]
```diff
@@ -65,13 +65,18 @@ import javax.ws.rs.GET;
...
 
 @Path("/hello")
 public class TracedResource {
 
+    private static final Logger LOG = LoggerFactory.getLogger(TracedResource.class);
```
Comment (gsmet) [inline docs/src/main/asciidoc/opentracing.adoc]: Please check that line too.

**Review by gsmet — CHANGES_REQUESTED** [review]

**Comment by pavolloffay** [comment]
@gsmet changed to jboss logging.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks, merged!

**Comment by pavolloffay** [comment]
Could you merge also https://github.com/quarkusio/quarkus-quickstarts/pull/485?

**Pull request merged**